src/Codegen/Generator.hack

```diff
@@ -144,7 +144,10 @@ class Field {
         $hb->addLine('return new GraphQL\\FieldDefinition(')->indent();
 
         // Field return type
-        $type_info = output_type($this->reflection_method->getReturnTypeText());
+        $type_info = output_type(
+            $this->reflection_method->getReturnTypeText(),
+            $this->reflection_method->getAttributeClass(\Slack\GraphQL\KillsParentOnException::class) is nonnull,
```

Comment:
lol this name 🤣

Reply:
This is actually the name we use internally, but like I said above it might be a good idea to change it to something less scary -- non-nullable fields are much less dangerous if they are used properly from the beginning. We had to discourage them mostly because we support lots of old clients with old assumptions.

I opened #42 for possible follow-up